Makefile

```diff
@@ -86,6 +86,7 @@ setup_frontend: $(WEBAPP)/.env
 	cd $(WEBAPP) && pnpm i
 
 setup: $(AZC) $(WZC) $(ZENROOM) $(ZENCODE) $(PB) setup_frontend ## 📦 Setup the project
+setup_arm: $(AZC) $(WZC) $(PB) setup_frontend
```

Comment:
If you need different architectures don't change the logic to be system wide... Instead add specific logic to be consistent and have (local installed) zenroom version with your architecture. I'm sure you'll find online plenty of examples for the os arch differentiation on makefiles, maybe also on other repos like starters

Reply:
I'll remove for now, and make an issue on starters